processor/attributesprocessor/attribute_hasher.go

```diff
@@ -58,6 +59,7 @@ func SHA1AttributeHahser(attr *tracepb.AttributeValue) *tracepb.AttributeValue {
 
 	var hashed string
 	if len(val) > 0 {
+		// #nosec
```

Comment:
I recommend adding explanations in the comments. In the case of unsafe pointer above it's more obvious, but not here.